ibm/flex/structures.go

```diff
@@ -3348,7 +3348,7 @@ func FlattenSatelliteHosts(hostList []kubernetesserviceapiv1.MultishiftQueueNode
 }
 
 func FlattenWorkerPoolHostLabels(hostLabels map[string]string) *schema.Set {
-	mapped := make([]string, len(hostLabels))
+	mapped := make([]string, len(hostLabels)-1)
```

Comment:
Even if user doesn't provide operating_system we always get "os" label ?

Reply:
Correct. The service will automatically assign an OS label based on the operating system of the worker pool. Current options are RHEL7, RHEL8, and RHCOS.